package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@willowtreeapps/rokul-runnings",
-  "version": "0.1.5",
+  "version": "0.2.0",
```

Comment:
Does this have breaking changes? might want to bump the major version, and add -beta or something. idk it's up to you

Reply:
May have some breaking changes, especially because of the removal of the WebDriverServer.

Buuuut I was always under the impression that versions under 1.0.0 would be considered "in development" and not really need to advertise breaking changes. Updated the minor version because it was a major change in the alpha version?

I think I'm gonna leave it at 0.2.0. After we've "fully" released (aka solidified things to a 1.0.0 release), we'll obviously need to advertise these things

Reply:
Cool sounds good just wanted to throw it out there